org.scala-ide.sdt.debug/src/org/scalaide/debug/internal/model/ScalaDebugTarget.scala

```diff
@@ -158,8 +166,14 @@ abstract class ScalaDebugTarget private(val virtualMachine: VirtualMachine,
   @volatile
   private var threads = List[ScalaThread]()
 
+  @volatile
+  private var isDuringHcr: Boolean = false
+  private[internal] def isPerformingHotCodeReplace: Boolean = isDuringHcr
+  private[internal] def isPerformingHotCodeReplace_=(isPerforming: Boolean) = { isDuringHcr = isPerforming }
```

Comment:
It seems to me I was using different access modifiers for a while. Currently it doesn't make sense to have a getter and a setter. Thanks.